en/guide/overview.md

```diff
@@ -1,13 +1,56 @@
 ## Overview
 
-The diagram below shows the main components of a system that is running the *Camera Streaming Daemon* on a Linux companion computer.
+The diagram below shows the main components of a "typical" system that is running the *Camera Streaming Daemon* on a Linux companion computer.
+CSD connects to multiple cameras that are attached to the linux computer. 
+It provides access to them via the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html) and RTSP video streams. It can also advertise available RTSP streams using Avahi. 
 
 ![Camera streaming daemon overview](../../assets/camera_streaming_daemon_overview.png)
 
-The server:
+In the diagram above the GCS requests for camera actions are forwarded by PX4 - CSD can also take MAVLink requests directly, and may do so in different configurations.
 
-* Scans and attaches to all compatible cameras. Cameras that support the [Video4Linux (V4L2) API](https://linuxtv.org/downloads/v4l-dvb-apis/uapi/v4l/v4l2.html) work out of the box (the server can be extended to support other camera protocols/APIs).
-* Advertises and shares RTSP video streams for all connected cameras. These can be consumed by the GCS and other video player services.
-* Captures and stores/shares video and still images for all connected cameras via the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html). GCS requests for camera actions are forwarded by PX4. 
 
-> **Note** The *Camera Streaming Daemon* also integrates with Gazebo, providing a simulated camera backend. In this case the diagram/configuration can be slightly different as the GCS can communicate directly with the server (there is no need for messages to be forwarded by PX4).
\ No newline at end of file
+## Key Features
+
+CSD supports the following key features:
+
+* Automatically attaches [compatible cameras](#supported_cameras) connected to the Linux computer.
+* RTSP video streaming from *all* connected cameras (for consumption by GCS or other video players).

```

Comment:
Is "all" correct? Specifically I am saying that in this section that you can have as many streams as cameras, but that you'll only be able to support 5 cameras (so far) for mavlink)

Reply:
> Specifically I am saying that in this section that you can have as many streams as cameras, but that you'll only be able to support 5 cameras (so far) for mavlink)

Thats true.

Video Capture is still not supported.

Reply:
> Video Capture is still not supported.

So video capture not even supported for MAVLink?

Reply:
Yes, thats right. I have plans to add it. When added, it will work with or without MAVLink